sound/soc/bcm/Kconfig

```diff
@@ -85,3 +85,12 @@ config SND_BCM2708_SOC_ADAU1977_ADC
 	select SND_SOC_ADAU1977_I2C
 	help
 	  Say Y or M if you want to add support for ADAU1977 ADC.
+
+config SND_BCM2708_SOC_FE_PI_AUDIO
+	tristate "Support for Fe-Pi-Audio"
+	depends on SND_BCM2708_SOC_I2S || SND_BCM2835_SOC_I2S
+	select SND_SOC_SGTL5000
+	select REGULATOR
```

Comment:
We don't use regulators in the Pi kernel; try not defining these CONFIG setting - you may find that it just works.